File: HB/pysrc/프로그래머스/레벨0/Day24/이진수_더하기.py
def solution(bin1, bin2):
    answer = ''
    b = 0
    bin = str(int(bin1)+int(bin2))[::-1]
    for i in range(len(bin)):
        a = ((int(bin[i])) + b)%2
        answer += str(a)
        b = (int(bin[i])+b)//2
        a=0
        if i == len(bin)-1 and b == 1:
            answer += '1'
    return answer[::-1]

File: SB/src/프로그래머스/레벨0/Day22/이진수_더하기.java
package 프로그래머스.레벨0.Day22;

public class 이진수_더하기 {
    class Solution {
        public String solution(String bin1, String bin2) {
            int a = Integer.parseInt(bin1, 2);
            int b = Integer.parseInt(bin2, 2);

            String answer = Integer.toBinaryString(a+b);
            return answer;
        }
    }
}
